routes/items.js

```diff
@@ -18,14 +23,32 @@ router.post("/", (req, res, next) => {
 });
 
 router.put("/:id", (req, res, next) => {
-  // TODO: complete writing this route handler
-  res.status(501).send({ error: 'route not yet implemented' });
+
+  const { id, field } = req.body
+  const updatedItem = {
+    id: id,
+    field: field
+  }
+  const theItem = itemData.updateById(req.params.id, updatedItem)
```

Comment:
No need for the variable assignment on line 28. This could just be 
`const theItem = itemData.updateById(req.params.id, req.body)`